server/routes/allocation-dashboard/handlers/activitiesDashboard.ts

```diff
@@ -13,7 +13,7 @@ export default class ActivitiesRoutes {
   GET = async (req: Request, res: Response): Promise<void> => {
     const { user } = res.locals
 
-    const activities = await this.activitiesService.getActivities(user).then(act =>
+    const activities = await this.activitiesService.getActivities(true, user).then(act =>
```

Comment:
I've noticed that activities with a start date in the future no longer show for allocation – do you know if this is intentional and whether this should be false?

Reply:
Hmm that is a good point ...